README.md

```diff
@@ -47,6 +47,89 @@ test logs from over-writing eachother.
 
 [config-wiki]: https://github.com/spekt/testlogger/wiki/Logger-Configuration
 
+### Junit Output Notes
+
+The xml reports will have standard output (stdout) and standard error (stderr) from the test output. Stdout is logged on a per-testcase basis, but stderr
```

Comment:
This section looks a bit incomplete. The beginning trails off and parts of mstest seems missing. I think the examples are helpful, but lets put them in another file for just the standard and error out. 

Since you are pointing out that failed tests behave differently can we just show all the options. Just 2 test methods should be fine.
*Passing Test
    * std out 
    * error out
*Failing Test
    * std out 
    * error out

I think it would be fine to have the docs show more complete examples in a `details` tag if its getting too long.

In this file, lets have a short section after `### Customizing Junit XML Contents` which points out that this stuff varies per testing framework, and links to all the details.